tests/Wallabag/ApiBundle/WallabagApiTestCase.php

```diff
@@ -31,12 +31,22 @@ protected function setUp(): void
         $this->client = $this->createAuthorizedClient();
     }
 
+    /**
+     * @return KernelBrowser
+     */
+    protected function createUnauthorizedClient()
```

Comment:
Why not?
```suggestion
    protected function createUnauthorizedClient(): KernelBrowser
```

Reply:
Mainly to be consistent with createAutorizedClient.
I think the move to native return type deserve its own work.
But I can do the change if you want in the name of continuous improvement :)

Reply:
No big deal 😉